contracts/examples/SimpleInterestTermsContract.sol

```diff
@@ -50,6 +50,10 @@ contract SimpleInterestTermsContract is TermsContract {
     // decimal values (such as interestRate) with fixed point values
     // scaled up by a factor of 10000 -- e.g. 10.342% => 1034250
     uint public constant INTEREST_RATE_SCALING_FACTOR = 10 ** 4;
+    // The number of orders of magnitudes an interest rate (denominated in %)
+    // must be scaled down in order to multiply it by a principal
+    // amount and produce the correct interest amount.
+    uint public constant PERCENTAGE_SCALING_FACTOR = 10 ** 2;
```

Comment:
Given that this is a clear area of confusion, perhaps documenting an example calculation might make sense.

Reply:
Likewise on board with this.